network/local.go

```diff
@@ -251,9 +265,10 @@ func (lc *LocalConn) Type() ConnType {
 	return Local
 }
 
-// connQueue is a struct that managers the message queue of a LocalConn.
-// Messages are pushed and retrieved FIFO style.
+// connQueue manages the message queue of a LocalConn.
+// Messages are pushed and retrieved in a FIFO.
```

Comment:
FIFO is not an object, but a method of ordering elements. A FIFO queue is an object. `...in a FIFO queue`